src/index.html

```diff
@@ -7,8 +7,25 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <a href=""><img src="images/logo.png" alt="" class="img_header"></a>
+
+      <nav class="nav">
+        <ul class="ul_header">
+          <li class="li_header"> <a href="" class="a__li">Apple</a></li>
```

Comment:
Need to fix it everywhere
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="" class="nav__link">Apple</a>
          </li>
          
```